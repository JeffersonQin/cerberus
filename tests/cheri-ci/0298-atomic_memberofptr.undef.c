struct T {
  int x;
};

int *p;
_Atomic(struct T) *st;

int main(void)
{
  p = &(st->x); // this is allowed
  *p; // this is undefined (ยง6.5.2.3#5)
}
