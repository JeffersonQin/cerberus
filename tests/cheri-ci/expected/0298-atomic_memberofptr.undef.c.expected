Undefined {ub: "UB043_indirection_invalid_value", stderr: "", loc: "<11:3--11:5>"}
